docs/source/hosts/evernode-diagnostics.md

```diff
@@ -101,3 +101,38 @@ Failed to retrieve the latest version data.
 ## 14. Re-config failure
 - If the `evernode config` command failed in any reconfiguration, retry executing it again.
 - If retry doesn't help change the value back to the original using the `evernode config` and change back to the new value again using the same command.
+
+## 15. Handling Reputation Assessment Observations
+
+### a. Issues with Reputation Sending Every Other Hour.
+- Typically, a host that registers for a reputation assessment sends the reputation scores at the end of the assessment period.
+- If the host is part of a "bad universe" (a group of underperforming hosts), the reputation contract may fail to execute correctly or reach consensus.
+- As a result, the host may be unable to send scores for that assessment round.
+- This will cause the Evernode Reputation Hook to reject the transaction, even if the host was registered for that moment.
+- Consequently, the host will not be registered for the next moment, causing it to miss that assessment as well.
+- As the universe assignment is an random process, we do not have a control over that, however the model always try to omit mal functioning nodes.
```

Comment:
malfunctioning

Reply:
Another point would be.
This could also mean that you are a bad actor. When there's a reputation running on your machine (You can check by `evernode list` command).
- Check HotPocket it's logs by `cat /home/<user>/<name>/log/hp.log` (Replace `user` and `name` respectively from the output of `evernode list`) and see it's completing consensus rounds by `****Ledger created****` log
  - If it's reporting `Not enough peers` check whether your domain name is correct and peer ports are reachable by outside.
- Check contract logs by `cat /home/<user>/<name>/log/contract/rw.stdout.log` and `cat /home/<user>/<name>/log/contract/rw.stderr.log`
  - `rw.stdout.log` should print logs of hash file creation and JSON containing the received scores against the peer public keys in the cluster. It will forcefully terminate if your host lacks minimum requirements. (Let's refer min req section in evernode-host page here, and also add a point there that min requirement per instance should be `mentioned min req / 3 (min to receive rewards)` to be able to run a reputation contract properly)